server.py

```diff
@@ -45,21 +48,26 @@ def write_file():
             os.makedirs(app.config['UPLOAD_FOLDER'])
         file.save(os.path.join(app.config['UPLOAD_FOLDER'], file_uuid))
         metadata.update_file_stored(file_uuid, app.config['HOST'])
+        logger.log(filename, ip_address, app.config['HOST'], 'WRITE', 201, os.path.getsize(file_path))
         return file_uuid, 201
+    logger.log(filename, ip_address, app.config['HOST'], 'WRITE', 500, -1)
```

Comment:
This might be more of a `400` error, not a `500` error because the user failed to provide a file.